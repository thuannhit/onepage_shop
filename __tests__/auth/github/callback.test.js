import React from 'react';
import renderer from 'react-test-renderer';
import Router from 'next/router';
import Callback from '../../../pages/auth/github/callback';

describe('Github Callback Page', () => {
  it('matches snapshot', () => {
    const mockedRouter = { push: () => {} };
    Router.router = mockedRouter;
    const component = renderer.create(<Callback accessToken="sag" />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
