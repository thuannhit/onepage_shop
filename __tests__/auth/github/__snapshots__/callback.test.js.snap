// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Github Callback Page matches snapshot 1`] = `null`;
